Doesn't work with Python 3.7

Hi. I'm using jip 0.9.13 with Python 3.7.1.
When I attempt to import `jip.repository`, python throws a syntax error.

```
koalanlp/Util.py:4: in <module>
    from jip import commands, logger
../../../virtualenv/python3.7.1/lib/python3.7/site-packages/jip/__init__.py:33: in <module>
    from jip.repository import repos_manager
../../../virtualenv/python3.7.1/lib/python3.7/site-packages/jip/repository.py:43: in <module>
    from jip.util import DownloadException, download, download_string, get_virtual_home
E     File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jip/util.py", line 45
E       def download(url, target, async=False, close_target=False, quiet=True):
E                                     ^
E   SyntaxError: invalid syntax
```

It seems that the argument name "async" is the problem, since that is newly introduced reserved keywords in Python 3.7.
So, could we change and update jip package to eliminate keyword names like "async", "await"?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,19 +1,25 @@
 language: python
-sudo: false
-python:
-- '2.6'
-- '2.7'
-- '3.4'
-- '3.5'
 matrix:
   include:
-  - python: 2.7
+  - python: '2.6'
+  - python: '2.7'
+    before_script:
+      - pip install flake8
+      - flake8 . --count --select=E9,F63,F72,F82 --show-source --statistics
+  - python: '3.5'
+  - python: '3.6'
+  - python: '3.7'
+    dist: xenial  # required for Python >= 3.7
+    before_script:
+      - pip install flake8
+      - flake8 . --count --select=E9,F63,F72,F82 --show-source --statistics
+  - python: '2.7'
     env: JYTHON=org.python:jython-installer:2.7.0
 before_install:
 - if [ -n "$JYTHON" ]; then ./ci/before_install_jython.sh; source $HOME/myvirtualenv/bin/activate
   ;fi
 install:
-- if [[ $TRAVIS_PYTHON_VERSION == 2.6 ]]; then pip install unittest2; fi
+- if [[ $TRAVIS_PYTHON_VERSION == "2.6" ]]; then pip install unittest2; fi
 - pip install -e .
 script:
 - python test/testsuite.py

--- a/jip/repository.py
+++ b/jip/repository.py
@@ -29,6 +29,7 @@ import os
 import locale
 import shutil
 import stat
+import sys
 import time
 import hashlib
 try:
@@ -287,7 +288,7 @@ class MavenHttpRemoteRepos(MavenRepos):
             hasher = hashlib.sha1()
 
         buf_size = 1024*8
-        file_to_check = file(filepath, 'r')
+        file_to_check = open(filepath, 'r')
         buf = file_to_check.read(buf_size)
         while len(buf) > 0:
             hasher.update(buf)

--- a/jip/util.py
+++ b/jip/util.py
@@ -42,10 +42,10 @@ BUF_SIZE = 4096
 class DownloadException(Exception):
     pass
 
-def download(url, target, async=False, close_target=False, quiet=True):
+def download(url, target, non_blocking=False, close_target=False, quiet=True):
     import requests
     ### download file to target (target is a file-like object)
-    if async:
+    if non_blocking:
         pool.submit(url, target)
     else:
         try:

